templates/default/rdrs-start.sh.erb

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env sh 
 
+
+<% if node['ndb']['rdrs']['containerize'].eql? "true" -%>
+
+rm -f <%= node['ndb']['log_dir'] %>/rdrs.cid 
+docker run --rm  \
+  --name rdrs-<%= node['ndb']['version'] %>.service  \
+  --cidfile <%= node['ndb']['log_dir'] %>/rdrs.cid \
+  --mount type=bind,source=<%= node['ndb']['root_dir'] %>/rdrs_config.json,target=<%= node['ndb']['root_dir'] %>/rdrs_config.json,readonly  \
+  --mount type=bind,source=<%= @crypto_dir %>,target=<%= @crypto_dir %>,readonly  \
+  --network=host \
+  rdrs:<%= node['ndb']['version'] %> \
+  /srv/hops/mysql/bin/rdrs -config=<%= node['ndb']['root_dir'] %>/rdrs_config.json
```

Comment:
The pattern we usually follow is to mount the logs directory so the logs are available outside the container and are later picked up by filebeat. Here the logs are only accessible from the container.

Also, I think, due to the `--rm` flag if you stop the container, the logs are lost I guess

Reply:
fixed